### DISCUSSION TITLE
Various issues

### DISCUSSION THREAD

**Discussion by Kistelini** [joeycastillo/Sensor-Watch#274] [General]
Hello,

I noticed various issues over time with my Sensor Watch:

- repeating alarms get cleared after random times (alarm_face)
- timers sometimes reset to zero before playing an alarm (countdown_face)
- sometimes the watch doesn't display anything but the red LED light; after a while it shows the correct time again

I rebuilt the firmware from the current main branch with some different faces, but still
- repeating alarms got cleared after two days
- hourly chime tune only works sometimes (didn't enable that before)
- temperature log lost all data points this morning (didn't enable thermistor_logging_face before)

I'm a bit lost because if those were software issues I surely wouldn't be the first one to notice those. Does anyone have any advice on how to debug this?

**Comment by wryun** [comment]
https://github.com/joeycastillo/Sensor-Watch/issues/275 might be the hourly chime thing.

The others I haven't seen... it sounds like the watch is being reset, which AFAIK would clear RAM but not the clock. This would explain why you lose settings/memory (i.e. alarms, timers, etc.).

So maybe a hardware fault or some kind of power issue (e.g. poor battery connection or just poor battery). A question for @joeycastillo , I think.

**Reply by Kistelini** [reply]
Thanks for the hint with the battery, @wryun! I reassembled the watch and replaced the double sided tape under the battery and since then everything works fine.
